aidatlu/constellation/README.md

```diff
@@ -0,0 +1,112 @@
+The software establishes a connection to the hardware and allows easy configuration of different trigger setups.
+Information over each individual trigger signal is saved in a compressed and human-readable HDF5 format.
+
+The satellite to connect the AIDA-2020 TLU to the [Constellation](https://constellation.pages.desy.de/) control and data acquisition framework.
+
+
+## Building
+
+After installing [IPbus](https://ipbus.web.cern.ch/doc/user/html/software/install/compile.html), also install the Python bindings and possible constellation requirement from the top file in the [Aida-TLU](https://github.com/SiLab-Bonn/aidatlu) repository.
+
+```bash
+pip install .[constellation]
+```
+
+A more detailed description of the prerequisite can also be found [here](https://silab-bonn.github.io/aidatlu/Introduction.html#installation).
+
+## Usage
+
+Add the cactus library path:
+
+```bash
+export LD_LIBRARY_PATH=<install_location>/lib
+```
+
+You also need to start the control hub:
+```bash
+/ipbus-software/controlhub/scripts/controlhub_start
+```
+
+Finally, start the satellite with:
+```bash
+SatelliteAidaTLU
+```
+
+The AIDA-TLU satellite can be tested using a
+TLU mock. By default, the hardware will be used, but the mock can be selected using an environmental variable.
+
+```bash
+TLU_MOCK=True SatelliteAidaTLU
+```
+
+## Parameters
+
+| Configuration | Description | Type | Default Value |
+|-----------|-------------|------| ------|
+| `internal_trigger_rate` | Generates internal trigger with a given frequency given in Hz | Integer | None |
+| `dut_interfaces` | Specify the operation mode of the DUT interface (`aida`, `eudet`, `aidatrig`) given in a list. | List | None |
+| `trigger_threshold` | Threshold setting of each individual trigger input channel given in V | List | None |
+| `trigger_inputs_logic` | Trigger Logic configuration accept a python expression for the trigger inputs. The logic is set by using the variables for the input channels 'CH1', 'CH2', 'CH3', 'CH4', 'CH5' and 'CH6' and the Python bitwise operators `and`, `or`, `not` and so on. Don't forget to use brackets... | String | None |
+| `trigger_polarity` | TLU can trigger on a rising or falling edge. | String | None |
+| `trigger_signal_stretch` | Stretches each individual trigger input by a given number of clock cycles (corresponds to 6.25ns steps) | List | None |
+| `trigger_signal_delay` | Delays each individual trigger input by a given number of clock cycles (corresponds to 6.25ns steps) | List | None |
+| `enable_clock_lemo_output` | Enable the LEMO clock output. | String | None |
+| `pmt_power` | Sets the four PMT control voltages in V | List | None |
+| `save_data` | Enables the creation of output data files. | String | None |
+| `output_data_path` | Specify a custom output data path to save the data to. If no path provided the TLU uses a default output folder. | String | None |
```

Comment:
"If no path is provided," and specify what the default is? Current directory? Relative to file?